docs/implementation-references/getting-started-guide.md

```diff
@@ -133,15 +133,97 @@ The build stage generates artifacts and stores them in `component-samples/demo`
 
 ### Key Generation for FDO Server-side Containers
 
-Run the below commands to generate all the required keys for the Server-side containers.
+1. Generating demo certificate authority KeyPair and certificate
+
+    ```shell
+      bash demo-ca.sh
+    ```
+
+    **NOTE**: Configure the properties of `demo-CA` by updating `root-ca.conf`.
+
+2. Generating Server and Client Keypair and certificates.
+
+    ```shell
+    bash web_csr_req.sh
+    bash user_csr_req.sh
+    ```
+
+    **NOTE**: Both Server and Client certificate are signed by the previously generated demo-CA. Moreover, we can configure the properties of Server and Client certificate by updating `web-server.conf` and `client.conf` respectively. [Learn how to configure Server and Client Certificates.](#specifying-subject-alternate-names-for-the-webhttps-self-signed-certificate)
```

Comment:
both server and client certificate -> certificates
all places need to be plural